features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -456,6 +457,7 @@
   "resource_access_control.scim2_me_post_method.keep_resource": true,
 
   "user.association.enable_for_federated_users": false,
+  "user.enable_per_user_functionality_locking": false,
```

Comment:
Shouldn't we have this enabled by default?

Reply:
To make use of the per-user functionality management feature, the users need to enable configuration management in their deployment. Since, there are certain prerequisites to enable configuration management (adding db tables), thought of not enabling by default.
@pulasthi7 WDYT?

Reply:
In master, all the tables are shipped by default for all database types since they are now used by other features as well.
Are there any other reasons not to enable this feature by default?

Reply:
enabled per-user functionality management feature by default with https://github.com/wso2/carbon-identity-framework/pull/3068/commits/3641437ebf81d2b4bbc1d7c4df7ac1ac01f8b110

Reply:
Shall we update the PR description? It says the default is false.

Reply:
Resolved.